test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/RandomOps.java

```diff
@@ -56,11 +58,22 @@ public static void main(String... args) {
 		return List.of(
 				new HapiApiSpec[]{
 						freezeWorks(),
+//						getAccountInfoRetryWorks()
 //						createThenTransferThenUpdateDeleteThenUpdate()
 				}
 		);
 	}
 
+	private HapiApiSpec getAccountInfoRetryWorks() {
+		return defaultHapiSpec("GetAccountInfoRetryWorks")
+				.given()
+				.when()
+				.then(
+						getAccountInfo("0.0.2")
```

Comment:
So does this go into an infinite loop? ♾️ 😄

Reply:
It is. It's better to have a check for retry times.

Reply:
Yup, added just for testing purposes. Will remove this before merging. 
@tinker-michaelj should we have a limit in both hapiTxnOp and hapiQuryOp for precheck retries ?

Reply:
@anighanta I'm not sure what might break if you set a _default_ limit. 😁 

But you could add an optional limit parameter, if you want.